tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -455,6 +455,8 @@ prepare_overlay()
 # provided as argument
 setup_rootfs()
 {
+	local target="linux-musl"
```

Comment:
I know it's not exactly your fault, but maybe it could be fixed here ...

What about building with glibc instead musl?
We'd be better having something you can easily set (as we do with the agent code).

Reply:
Or maybe not, ignore my comment and I'll file an issue for this one. :-)

Reply:
@fidencio are you suggesting defaulting to glibc instead of musl ?

Reply:
No, no, I'm suggesting taking into consideration the value set for "LIBC", defaulting to that instead of hardcoding `linux-musl`.

In any case, this bit is not related to this PR, it was like this before and I think we'd be better treating my suggestion in a different PR (I've opened https://github.com/kata-containers/kata-containers/issues/682 to track that).

Reply:
That makes me wonder if this is a ppc64le issue, or a specific setup.

In other words, shouldn't we check if we have gnu or if we have musl, and act accordingly.

Asking because on my x86_64 Fedora32, I believe I run into the same issue if I don't specify `LIBC=gnu`.

Reply:
> No, no, I'm suggesting taking into consideration the value set for "LIBC", defaulting to that instead of hardcoding `linux-musl`.
> 
> In any case, this bit is not related to this PR, it was like this before and I think we'd be better treating my suggestion in a different PR (I've opened #682 to track that).

yes @fidencio , this approach would be better. Rather than having another variable, the existing one(`LIBC`) could be used. To confirm, were you mentioning about something like this - `linux-${LIBC}`?

Reply:
Looking at the rust agent Makefile, what id needs is a triple like: `$(ARCH)-unknown-linux-$(LIBC)` (see: https://github.com/kata-containers/kata-containers/blob/2.0-dev/src/agent/Makefile#L47), so using linux-$(LIBC) may be enough for that.

We'll need to test and ensure it works with both cases.

Reply:
Thanks @fidencio , will make the changes and test it out.

Reply:
>Looking at the rust agent Makefile, what id needs is a triple like: $(ARCH)-unknown-linux-$(LIBC)

@fidencio Technically it's a quadruple :smiley: